File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void pyramid(int height);

int main(void)
{
    int input = get_int("Height: ");
    pyramid(input);
}

void pyramid(int height) {

    for (int i = 0; i < height; i++) {

        for (int j = 0; j < 5; j++ ) {

            printf("#");

    }
printf("\n");
}
}

File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

spaces = height - 1

for _ in range(height):

    print(" " * spaces, end="")

    print("#" * (height - spaces), end="")

    print("  ", end="")

    print("#" * (height - spaces), end="")

    print("")
    spaces -= 1
